lib/lintSource.js

```diff
@@ -84,7 +81,7 @@ module.exports = function lintSource(stylelint, options = {}) {
 					stylelint: stylelintResult,
 				});
 
-				return lintPostcssResult(stylelint, stylelintPostcssResult, config).then(
+				return lintPostcssResult(stylelint._options, stylelintPostcssResult, config).then(
```

Comment:
`stylelint._options` was the only part of the `stylelint` object that was being used, so I changed `lintPostcssResult` to accept the options object directly.